tests/integration/deploy-operator.sh

```diff
@@ -0,0 +1,87 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Exit immediately for non zero status
+set -e
+# Check unset variables
+set -u
+# Print commands
+set -x
+
+install-operator-k8s() { # installs istio-operator on kubernetes
+    local ROOT
+    ROOT="$(git rev-parse --show-toplevel)"
+    local TAG
+    TAG="$(git rev-parse HEAD)"
+    local NS="${NS:-openshift-operators}"
+    # var name IMAGE is in kind provisioner
+    local OPERATOR_IMAGE="${OPERATOR_IMAGE:-localhost:5000/istio-operator-integ}:${TAG}"
+    local ISTIO_REPO="${ISTIO_REPO:-quay.io/maistra-dev}"
+    local ISTIO_TAG="${ISTIO_TAG:-latest}"
+    local ISTIO_CNI_IMAGE_NAME="istio-cni-ubi8-integ"
+    local PILOT_IMAGE_NAME="pilot-ubi8-integ"
+    local PROXY_IMAGE_NAME="proxyv2-ubi8-integ"
+    # var name VERSION is in Makefile
+    local VERSION="2.3.0"
+    local OSSM_20_TAG="2.0.11"
+    local OSSM_21_TAG="2.1.5"
+    local OSSM_22_TAG="2.2.4"
+
+    echo "--------------------------------"
+    echo "Installing operator"
+    echo "Using image $OPERATOR_IMAGE"
+    echo "--------------------------------"
+
+    kubectl get ns "$NS" >/dev/null 2>&1 || kubectl create namespace "$NS"
+
+    kubectl apply -f "${ROOT}/manifests-servicemesh/${VERSION}/servicemeshcontrolplanes.crd.yaml"
+    kubectl apply -f "${ROOT}/manifests-servicemesh/${VERSION}/servicemeshmemberrolls.crd.yaml"
+    kubectl apply -f "${ROOT}/manifests-servicemesh/${VERSION}/servicemeshmembers.crd.yaml"
+
+    sed -e "s/namespace: istio-operator/namespace: $NS/g" "${ROOT}/deploy/src/rbac.yaml" | kubectl apply -n "$NS" -f -
+    sed -e "s/namespace: istio-operator/namespace: $NS/g" "${ROOT}/deploy/src/serviceaccount.yaml" | kubectl apply -n "$NS" -f -
+    sed -e "s/namespace: istio-operator/namespace: $NS/g" "${ROOT}/deploy/src/service.yaml" | kubectl apply -n "$NS" -f -
+
+    openssl req -x509 -newkey rsa:4096 -keyout /tmp/key.pem -out /tmp/cert.pem -sha256 -days 365 -nodes -subj "/CN=istio-operator" -addext "subjectAltName = DNS:maistra-admission-controller.$NS.svc"
+
+    kubectl create -n "$NS" secret tls maistra-operator-serving-cert --key=/tmp/key.pem --cert=/tmp/cert.pem
+    kubectl create -n "$NS" configmap maistra-operator-cabundle --from-file=service-ca.crt=/tmp/cert.pem
+
+    # shellcheck disable=SC1004
+    sed -e "s/^\(\s*\)\(serviceAccountName:\)/\1imagePullSecrets:\n\1- name: quay-pull-secret\n\1\2/g" \
+        -e "s/image: .*/image: ${OPERATOR_IMAGE//\//\\/}/g" \
+        -e "s/\${OSSM_OPERATOR_IMAGE}/${OPERATOR_IMAGE//\//\\/}/g" \
+        -e "s/\${OSSM_20}/${OSSM_20_TAG}/g" \
+        -e "s/\${OSSM_21}/${OSSM_21_TAG}/g" \
+        -e "s/\${OSSM_22}/${OSSM_22_TAG}/g" \
+        -e "s/\${OSSM_CNI_IMAGE}/${ISTIO_REPO//\//\\/}\/${ISTIO_CNI_IMAGE_NAME}:${ISTIO_TAG}/g" \
```

Comment:
well, that's a placeholder cni image. I pushed it in quay. Otherwise, we don't have a release tag , do we ?
https://quay.io/repository/maistra-dev/istio-cni-ubi8-integ?tab=tags